pkg/apis/intelligence/v1alpha1/types.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2022 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+type JobType string
+
+const (
+	NPRecommendationJobInitial    JobType = "Initial"
+	NPRecommendationJobSubsequent JobType = "Subsequent"
```

Comment:
May I ask why we create const for JobType? And why JobType only?

Reply:
for testing and providing a template of where to define these const. more can be added when consumed.